extensions/wikia/ArticleVideo/ArticleVideo.i18n.php

```diff
@@ -4,8 +4,10 @@
 
 $messages['en'] = [
 	'articlevideo-watch' => 'watch • $1',
+	'articlevideo-related-video' => 'Related video',
 ];
 
 $messages['qqq'] = [
-	'articlevideo-watch' => '"Watch" string with a video length',
-];
\ No newline at end of file
+	'articlevideo-watch' => '"Watch" string before video time',
```

Comment:
`'articlevideo-watch' => 'watch • $1'` is untouched, so this description seems to not be true.